### DISCUSSION TITLE
Alexa -> Jarvis

### DISCUSSION THREAD

**Discussion by yashwankhede** [community/community#125517] [Programming Help]
Can we change the voice of Alexa to sound like a Jarvis? 🤔

**Comment by RohanHandore** [comment]
yes see the documentation for it :
https://github.com/orgs/community/discussions/125517

**Comment by RohanHandore** [comment]
https://developer.amazon.com/en-US/docs/alexa/documentation-home.html

visit this

**Comment by judahpaul16** [comment]
Build your own with [GPT Home](https://github.com/judahpaul16/gpt-home) on a Raspberry Pi and modify the default ALSA voice? Maybe check out [TextyMcSpeechy](https://github.com/domesticatedviking/TextyMcSpeechy) for an even more custom voice.

**Reply by judahpaul16** [reply]
If you wanna go hardcore: https://www.youtube.com/watch?v=bO-DWWFolPw

**Comment by Geapen1503** [comment]
You could try to put alexa audio output to a pc that have voicemod and return you the new audio, moded with jarvis voice
